.github/UPGRADE_PATH.md

```diff
@@ -101,21 +101,21 @@ Our directions are written for only 1 major version upgrade at a time, as we hav
     self?.proceedInWorkflow(order)
   }
   ```
-  
+
   ### The way you Test has Changed
   You used to be able to re-assign `proceedInWorkflow` to assert it was called with the args you expected, this has now slightly changed.
-  To get the *exact* behavior as before use `_proceedInWorkflow` to re-assign that closure. 
+  To get the *exact* behavior as before use `_proceedInWorkflow` to re-assign that closure.
   There's also `proceedInWorkflowStorage` which gives you the `AnyWorkflow.PassedArgs` used when `proceedInWorkflow` was called.
-  
```

Comment:
I use a Markdown linter in VSCode and it haaaates when these spaces exist 😂, glad to see this lol

Reply:
Yeah I could probably remove all of them, but my editor just removed the ones in the files I edited.